src/client/apps/edit/components/admin/components/appearances.tsx

```diff
@@ -1,50 +1,50 @@
+import { Box, Flex } from "@artsy/palette"
 import React from "react"
-import { Col, Row } from "react-styled-flexboxgrid"
-import AutocompleteListMetaphysics from "client/components/autocomplete2/list_metaphysics"
+import AutocompleteListMetaphysics from "../../../../../components/autocomplete2/list_metaphysics"
 
-export const AdminAppearances = props => {
+export const AdminAppearances = () => {
   return (
-    <div>
-      <Row>
-        <Col xs={6}>
+    <React.Fragment>
```

Comment:
Could we use the `<>...</>` syntax or we aren't there yet?

Reply:
tsx files support it